src/components/Moveable.vue

```diff
@@ -88,9 +88,12 @@ export default {
       ...this.$props,
       target: this.$el,
     });
-    EVENTS.forEach((event) => (
-      this.moveable.on(event, this.$emit.bind(this, event))
-    ));
+    EVENTS.forEach((event) => {
+      const kebabCaseEvent = event.replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2').toLowerCase();
+      this.moveable.on(event, this.$emit.bind(this, kebabCaseEvent));
+      // Backwards support for camelCase events
+      this.moveable.on(event, this.$emit.bind(this, event));
```

Comment:
I don't think backward support is needed. 
Events in `camelCase` are not working anyway. Remove this line and I'll merge it

Reply:
The current `camelCase` event are working properly. Removing this line would deprecate all projects using the code like the example below
```
<moveable
        @dragStart="dragStartHandle"
        @resizeEnd="resizeEndHandle"
>
...
</moveable>
```

because we wouldn't listen to `dragStart` anymore. I think we should support both ways (listen to events like `drag-start` and `dragStart`) at least for the next upcoming version (or even to the next major release).